### PR TITLE
Adds more Info button to Symbol Locations

### PR DESCRIPTION
**Pull request by antonrohr**
This changes the explanatory text in the Symbol Locations dialog and adds a more info button go/stadia-orbit-symbol-loading-explanatory-wording

screenshot: https://screenshot.googleplex.com/Bxzj9t4iNbvauCZ

### CODE REVIEW

File: src/ConfigWidgets/SymbolsDialog.cpp [hunk 1]
```diff
@@ -192,4 +194,14 @@ void SymbolsDialog::OnListItemSelectionChanged() {
   ui_->removeButton->setEnabled(!ui_->listWidget->selectedItems().isEmpty());
 }
 
+void SymbolsDialog::OnMoreInfoButtonClicked() {
+  QString url_as_string{
```
Comment (reichlfl) [inline src/ConfigWidgets/SymbolsDialog.cpp:198]: I thought we had some restriction on linking to Stadia Docs, and there was a reason why we're not doing it anywhere. I might mix that up though - @ronaldfw ?

Comment (ronaldfw) [reply src/ConfigWidgets/SymbolsDialog.cpp:198]: Yeah, we generally don't want to do that, with possible exceptions (we link to the Stadia community sites for feedback). I'll share more internally. 

Apart from the feedback links, I think we do not want to have links to documentation that is not generally accessible for all open source users.

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM

File: src/ConfigWidgets/SymbolsDialog.cpp [hunk 2]
```diff
@@ -20,6 +21,7 @@
 #include "ObjectUtils/SymbolsFile.h"
 #include "OrbitBase/Logging.h"
 #include "OrbitBase/Result.h"
+#include "Symbols/SymbolUtils.h"
```
[the remark this answers is not in the document]
Comment (antonrohr) [reply src/ConfigWidgets/SymbolsDialog.cpp]: no, artifact from earlier iteration. done

File: src/ConfigWidgets/SymbolsDialogTest.cpp [hunk 3]
```diff
@@ -46,6 +46,10 @@ TEST(SymbolsDialog, ConstructEmpty) {
   auto* list_widget = dialog.findChild<QListWidget*>("listWidget");
   ASSERT_NE(list_widget, nullptr);
   EXPECT_EQ(list_widget->count(), 0);
+
+  auto* more_info_button = dialog.findChild<QPushButton*>("moreInfoButton");
+  ASSERT_NE(more_info_button, nullptr);
+  EXPECT_EQ(more_info_button->text().toStdString(), "More information on symbol loading");
```
[the remark this answers is not in the document]
Comment (antonrohr) [reply src/ConfigWidgets/SymbolsDialogTest.cpp]: I agree. This is actually a leftover from an earlier iteration where there was actual functionality tested. The earlier idea was a button that would expand another box and the button text would change from "Show more info" to "Hide more info". Its removed now

**Pull request merged**